modules/replicator/src/main/java/org/apache/ignite/internal/replicator/CommandApplicationResult.java

```diff
@@ -19,24 +19,25 @@
 
 import java.util.concurrent.CompletableFuture;
 import org.apache.ignite.internal.hlc.HybridTimestamp;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * Replication command application result.
  */
 public final class CommandApplicationResult {
-    private final HybridTimestamp commitTs;
-    private final CompletableFuture<?> repFut;
+    private final @Nullable HybridTimestamp commitTs;
```

Comment:
It's not related to the patch, just some mix-ins, right?

Reply:
Just added Nullable annotations for clarity.

Reply:
I guess that was from https://github.com/apache/ignite-3/pull/4700